src/GovUk.Education.ExploreEducationStatistics.Content.Model/DataSetFileMeta.cs

```diff
@@ -11,31 +12,52 @@ public class DataSetFileMeta
     public List<string> GeographicLevels { get; set; } = new();
 
     [JsonConverter(typeof(TimeIdentifierJsonConverter))]
-    public TimeIdentifier TimeIdentifier { get; set; }
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public TimeIdentifier? TimeIdentifier { get; set; } // EES-4918 to remove
+
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public List<int>? Years { get; set; } = new(); // EES-4918 to remove
 
-    public List<int> Years { get; set; } = new();
+    public TimePeriodRangeMeta TimePeriodRange { get; set; } = null!;
 
     public List<FilterMeta> Filters { get; set; } = new();
 
     public List<IndicatorMeta> Indicators { get; set; } = new();
 }
 
+public class TimePeriodRangeMeta
+{
+    public TimePeriodRangeBoundMeta Start { get; set; } = null!;
+
+    public TimePeriodRangeBoundMeta End { get; set; } = null!;
+}
+
+public class TimePeriodRangeBoundMeta
+{
+    [JsonConverter(typeof(TimeIdentifierJsonConverter))]
+    public TimeIdentifier TimeIdentifier { get; set; }
+
+    public int Year { get; set; }
```

Comment:
Would change this to `string Period` as we've modelled it in the public API. 

We've purposefully done this because singular dates and date ranges are eventually coming down the pipeline ([EES-3109](https://dfedigital.atlassian.net/browse/EES-3109)). 

These new time periods are potentially going in the `time_period` column, so the current assumptions that we make all over the place this is numeric year are likely to be incorrect in the future. Modelling our meta summary as suggested helps prevent this being an issue later down the line.